frontend/server/controllers/UserController.php

```diff
@@ -2477,16 +2477,17 @@ public static function getCoderOfTheMonthDetailsForSmarty(
         if (!$isMentor) {
             return ['payload' => $response];
         }
-        $bestCoders = [];
         $candidates = CoderOfTheMonthDAO::calculateCoderOfMonthByGivenDate(
             $dateToSelect
         );
-        if ($candidates == null) {
-            $candidates = [];
-        }
-        foreach ($candidates as $candidate) {
-            unset($candidate['user_id']);
-            $bestCoders[] = $candidate;
+        $bestCoders = $candidates;
```

Comment:
```suggestion
        $bestCoders = [];
```

así te ahorras L2486.